content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -210,17 +181,59 @@ curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools
 .\PrepareNode.ps1 -KubernetesVersion {{< param "fullversion" >}} -ContainerRuntime containerD
 ```
 
+[Install `crictl` from the cri-tools package](https://github.com/kubernetes-sigs/cri-tools)
+which is required so that kubeadm can talk to the CRI endpoint.
+
 #### Run `kubeadm` to join the node
 
 Use the command that was given to you when you ran `kubeadm init` on a control plane host.
 If you no longer have this command, or the token has expired, you can run `kubeadm token create --print-join-command`
 (on a control plane host) to generate a new token and join command.
 
+{{% /tab %}}
+
+{{% tab name="Docker Engine" %}}
+
+#### Install Docker Engine
+
+Install the `Containers` feature
+
+```powershell
+Install-WindowsFeature -Name containers
+```
+
+Install Docker
+Instructions to do so are available at [Install Docker Engine - Enterprise on Windows Servers](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server#install-docker).
```

Comment:
BTW, this links to a page stating that
> Customers who want to install a container runtime on Windows server are encouraged to transition to either containerd, Moby, or the Mirantis Container Runtime. 

Not sure what we should cover. Maybe we should advise readers to install one of those 3 things and then continue? We can cite those as Microsoft's suggested runtimes for Windows servers.
I think `cri-dockerd` will work with Moby (Docker Engine without the trademarks) but I'm not sure - I don't actually use Docker very much.

Reply:
i'm not familiar with the Mirantis product line.

follow ups related to Windows nodes can be tracked in:
https://github.com/kubernetes/website/issues/31395